content/docs/04-clusters/03-edge/01.8-installer-image.md

```diff
@@ -0,0 +1,248 @@
+    ...
+    ...
+    INFO[2023-02-15T22:08:06Z] Creating squashfs...
+    DEBU[2023-02-15T22:08:06Z] Running cmd: 'mksquashfs /tmp/elemental-iso972415663/rootfs /tmp/elemental-iso972415663/iso/rootfs.squashfs -b 1024k -comp xz -Xbcj x86'
+    ISO image produced: 161394 sectors
+    Written to medium : 161424 sectors at LBA 48
+    Writing to '/cOS/p6os-custom.iso' completed successfully
+    ```
+
+
+6. The ISO file can be found in the root directory. Mount the ISO using a bootable USB drive, PXE server, or any other means to the bare metal appliances' primary drive. The installer will be flashed to the edge host hard disk, and the host will be shut down. The bare-metal edge host appliance is ready to be shipped to the edge location.
+
+
+<InfoBox>
+
+You can use several software tools to create a bootable USB drive, such as [balenaEtcher](https://www.balena.io/etcher). For PXE server, there are open-source projects such as [Fog](https://fogproject.org/download) or [Windows Deployment Services](https://learn.microsoft.com/en-us/windows/deployment/wds-boot-support) for Windows.
+
+</InfoBox>
+
+
+To deploy the device and register the device with Palette, refer to the [Register and Manage Edge Native Clusters](/clusters/edge/native) guide.
+
+
+## Validation
+
+You can validate that the ISO image is not corrupted by attempting to flash a bootable device. Most software that creates a bootable device will validate the ISO image before the flash process. 
+
+</Tabs.TabPane>
+
+<Tabs.TabPane tab="VMware" key="vmware">
+
+## VMware
+
+Create an Open Virtual Appliance (OVA) file from the base Palette Edge Installer ISO for VMware environments by injecting customized settings (user data) through a secondary drive. 
+
+## Prerequisites
+
+* mkisofs, or genisoimage, or similar ISO management software.
+
+
+* cdrtools or [wodim](https://cygwin.com/packages/summary/wodim.html) for Windows
+
+
+* Access to vSphere.
+
+## Create OVA Image
+
+
+1. Create a file called **user-data**. 
+
+  ```shell
+  touch user-data
+  ```
+
+2. Use the content from the template in the [Setup Device section](/clusters/edge/native#setupdevice) to add your site configuration. Make the necessary changes for your environment.
+
+
+2. Create an empty **meta-data** file:
+
+  ```
+  touch meta-data
+  ```
+
+3. Create a cloud init ISO using the following command.
+
+  MacOS/Linux:
+  ```
+  mkisofs -output ci.iso -volid cidata -joliet -rock user-data meta-data
+  ```
+
+  Windows:
+  ```
+  genisoimage -output ci.iso -volid cidata -joliet -rock user-data meta-data
+  ```
+This will generate an ISO file called ci.iso in the current directory.
+
+
+4. Upload the **ci.iso** file generated in the previous step to a datastore in vSphere using the vCenter console.
+
+
+5. Log in to [Palette](https://console.spectrocloud.com).
+
+
+6. Navigate to the left **Main Menu** and select **Clusters**. 
+
+
+7. Click on the **Edge Hosts** tab. 
```

Comment:
Karl, try combining this with the next step, as shown in line 199.

The optimum number of steps in a procedure is 7. Sometimes, you need more, but these steps combine well to reduce the number of overall steps.